### ISSUE TITLE
Horizontal scroll at the end position doesn't show part of line of editor.

### ISSUE THREAD

**Issue by Kvaz1r** [wxFormBuilder/wxFormBuilder#569]
wxFB version: build from source current master
OS: Win10

Picture:
![wxFB_1](https://user-images.githubusercontent.com/17313967/69947364-2c8a6880-14f6-11ea-97ec-a6e96b820dd5.jpg)

**Comment by sodevel** [comment]
This is most probably a problem of wxScintilla or wrong usage of that one, i see this issue in e.g. CodeBlocks as well. As workaround you can put the cursor in the too long line and use keyboard navigation to the line end, this updates the scrollbar length.

**Comment by Kvaz1r** [comment]
Yeah seems like wxScintilla issue. It's also possible move view via button on scroll bar.
